CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 0.13.0
+
+### Technical improvements
+
+- Integrated new rendering engine for Object & Lane Detection, Segmentation & AR
```

Comment:
I guess here and everywhere below we should have `Integrate` / `Add` etc

Reply:
yeah, fixed